components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.h

```diff
@@ -24,6 +24,21 @@
  Delegate for MDCBottomDrawerContainerViewController.
  */
 @protocol MDCBottomDrawerContainerViewControllerDelegate <NSObject>
+
+/**
+This method is called when the bottom drawer updates its value for scrollViewIsScrolledToBottom.
```

Comment:
we should probably indicate that this is only activated when trackingScrollView is set

Reply:
I renamed `scrollViewIsScrolledToBottom ` to `scrollViewIsScrolledToEndOfContent` so people wouldn't confuse it with the preexisting (and questionably named) `scrollViewIsDraggedToBottom` property, but failed to change it in some of the documentation, including this line.